### ISSUE TITLE
Zero is not allowed in range of sequence option specifications.

### ISSUE THREAD

**Issue by savagepanda** [commandlineparser/commandline#302]
I need a way to indicate that I don't accept any Value parameters.
I tried
[Value(0, Max=0)]
public IEnumerable<string> Items {get; set;}

but get this error.  Seems like there currently is no way to say you don't want any values.  min value for the Max property is 1.

**Comment by moh-hassan** [comment]
>Seems like there currently is no way to say you don't want any values

The only way to  say you don't want a value  is not passing the option in the commandline
All scalar and sequence options should have a value otherwise the parser raise an error.

Only switches (Boolean) can be passed without a value and it means the option is true.

**Comment by nemec** [comment]
Yes, as @moh-hassan says if you don't want any values you need to set it to boolean. If the flag is set, it will be true. If not, it will be false.

**Comment by konrad-kruczynski** [comment]
I think that @savagepanda meant that there is no way to get result with tag `NotParsed` when user specifies value arguments and we don't want any. IOW with this simple option class:
```
class Options
{
    [Option("op")]
    public bool Option {get; set; }
}
```
it is legal not only to issue `myprog.exe --op` at command line, but parser succeeds on `myprog.exe --op something` etc. as well. Even if there is no `[Value]` in the option class.
